app/assets/stylesheets/application.scss

```diff
@@ -14,6 +14,11 @@
  *= require_self
  *= require foundation_and_overrides
  */
+html,
+body {
+	height: 100%;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered font-family, height